TestPilosaHiccup fails randomly

It also makes `TestDelete` fail. Setting `GO_MAXPROCS=1` alleviates the problem.

```
=== RUN   TestPilosaHiccup
--- FAIL: TestPilosaHiccup (0.17s)
	driver_test.go:258: [0] values: [7a2f85497548f3db88542025e6f75908b8716c12-00 2346ad27d7568ba9896f1b7da6b5991251debdf2-00] location: 00 loc: 00 err: <nil>
	driver_test.go:258: [1] values: [7a2f85497548f3db88542025e6f75908b8716c12-01 2346ad27d7568ba9896f1b7da6b5991251debdf2-01] location: 01 loc: 01 err: <nil>
	driver_test.go:258: [2] values: [7a2f85497548f3db88542025e6f75908b8716c12-02 2346ad27d7568ba9896f1b7da6b5991251debdf2-02] location: 02 loc: 02 err: <nil>
	driver_test.go:258: [3] values: [7a2f85497548f3db88542025e6f75908b8716c12-03 2346ad27d7568ba9896f1b7da6b5991251debdf2-03] location: 03 loc: 03 err: <nil>
	driver_test.go:258: [4] values: [7a2f85497548f3db88542025e6f75908b8716c12-04 2346ad27d7568ba9896f1b7da6b5991251debdf2-04] location: 04 loc: 04 err: <nil>
	driver_test.go:258: [5] values: [7a2f85497548f3db88542025e6f75908b8716c12-05 2346ad27d7568ba9896f1b7da6b5991251debdf2-05] location: 05 loc: 05 err: <nil>
	driver_test.go:258: [6] values: [7a2f85497548f3db88542025e6f75908b8716c12-06 2346ad27d7568ba9896f1b7da6b5991251debdf2-06] location: 06 loc: 06 err: <nil>
	driver_test.go:258: [7] values: [7a2f85497548f3db88542025e6f75908b8716c12-07 2346ad27d7568ba9896f1b7da6b5991251debdf2-07] location: 07 loc: 07 err: <nil>
	driver_test.go:258: [8] values: [7a2f85497548f3db88542025e6f75908b8716c12-08 2346ad27d7568ba9896f1b7da6b5991251debdf2-08] location: 08 loc: 08 err: <nil>
	driver_test.go:258: [9] values: [7a2f85497548f3db88542025e6f75908b8716c12-09 2346ad27d7568ba9896f1b7da6b5991251debdf2-09] location: 09 loc: 09 err: <nil>
	driver_test.go:258: [10] values: [7a2f85497548f3db88542025e6f75908b8716c12-00 2346ad27d7568ba9896f1b7da6b5991251debdf2-00] location: 00 loc: 00 err: <nil>
	driver_test.go:258: [11] values: [7a2f85497548f3db88542025e6f75908b8716c12-01 2346ad27d7568ba9896f1b7da6b5991251debdf2-01] location: 01 loc: 01 err: <nil>
	driver_test.go:258: [12] values: [7a2f85497548f3db88542025e6f75908b8716c12-02 2346ad27d7568ba9896f1b7da6b5991251debdf2-02] location: 02 loc: 02 err: <nil>
	driver_test.go:258: [13] values: [7a2f85497548f3db88542025e6f75908b8716c12-03 2346ad27d7568ba9896f1b7da6b5991251debdf2-03] location: 03 loc: 03 err: <nil>
	driver_test.go:258: [14] values: [7a2f85497548f3db88542025e6f75908b8716c12-04 2346ad27d7568ba9896f1b7da6b5991251debdf2-04] location: 04 loc: 04 err: <nil>
	driver_test.go:258: [15] values: [7a2f85497548f3db88542025e6f75908b8716c12-05 2346ad27d7568ba9896f1b7da6b5991251debdf2-05] location: 05 loc: 05 err: <nil>
	driver_test.go:258: [16] values: [7a2f85497548f3db88542025e6f75908b8716c12-06 2346ad27d7568ba9896f1b7da6b5991251debdf2-06] location: 06 loc: 06 err: <nil>
	driver_test.go:258: [17] values: [7a2f85497548f3db88542025e6f75908b8716c12-07 2346ad27d7568ba9896f1b7da6b5991251debdf2-07] location: 07 loc: 07 err: <nil>
	driver_test.go:258: [18] values: [7a2f85497548f3db88542025e6f75908b8716c12-08 2346ad27d7568ba9896f1b7da6b5991251debdf2-08] location: 08 loc: 08 err: <nil>
	driver_test.go:258: [19] values: [7a2f85497548f3db88542025e6f75908b8716c12-09 2346ad27d7568ba9896f1b7da6b5991251debdf2-09] location: 09 loc: 09 err: <nil>
	driver_test.go:258: [20] values: [7a2f85497548f3db88542025e6f75908b8716c12-00 2346ad27d7568ba9896f1b7da6b5991251debdf2-00] location: 00 loc: 00 err: <nil>
	driver_test.go:258: [21] values: [7a2f85497548f3db88542025e6f75908b8716c12-01 2346ad27d7568ba9896f1b7da6b5991251debdf2-01] location: 01 loc: 01 err: <nil>
	driver_test.go:258: [22] values: [7a2f85497548f3db88542025e6f75908b8716c12-02 2346ad27d7568ba9896f1b7da6b5991251debdf2-02] location: 02 loc: 02 err: <nil>
	driver_test.go:258: [23] values: [7a2f85497548f3db88542025e6f75908b8716c12-03 2346ad27d7568ba9896f1b7da6b5991251debdf2-03] location: 03 loc: 03 err: <nil>
	driver_test.go:258: [24] values: [7a2f85497548f3db88542025e6f75908b8716c12-04 2346ad27d7568ba9896f1b7da6b5991251debdf2-04] location: 04 loc: 04 err: <nil>
	driver_test.go:258: [25] values: [7a2f85497548f3db88542025e6f75908b8716c12-05 2346ad27d7568ba9896f1b7da6b5991251debdf2-05] location: 05 loc: 05 err: <nil>
	driver_test.go:258: [26] values: [7a2f85497548f3db88542025e6f75908b8716c12-06 2346ad27d7568ba9896f1b7da6b5991251debdf2-06] location: 06 loc: 06 err: <nil>
	driver_test.go:258: [27] values: [7a2f85497548f3db88542025e6f75908b8716c12-07 2346ad27d7568ba9896f1b7da6b5991251debdf2-07] location: 07 loc: 07 err: <nil>
	driver_test.go:258: [28] values: [7a2f85497548f3db88542025e6f75908b8716c12-08 2346ad27d7568ba9896f1b7da6b5991251debdf2-08] location: 08 loc: 08 err: <nil>
	driver_test.go:258: [29] values: [7a2f85497548f3db88542025e6f75908b8716c12-09 2346ad27d7568ba9896f1b7da6b5991251debdf2-09] location: 09 loc: 09 err: <nil>
	driver_test.go:258: [30] values: [7a2f85497548f3db88542025e6f75908b8716c12-00 2346ad27d7568ba9896f1b7da6b5991251debdf2-00] location: 00 loc: 00 err: <nil>
	driver_test.go:258: [31] values: [7a2f85497548f3db88542025e6f75908b8716c12-01 2346ad27d7568ba9896f1b7da6b5991251debdf2-01] location: 01 loc: 01 err: <nil>
	driver_test.go:258: [32] values: [7a2f85497548f3db88542025e6f75908b8716c12-02 2346ad27d7568ba9896f1b7da6b5991251debdf2-02] location: 02 loc: 02 err: <nil>
	driver_test.go:258: [33] values: [7a2f85497548f3db88542025e6f75908b8716c12-03 2346ad27d7568ba9896f1b7da6b5991251debdf2-03] location: 03 loc: 03 err: <nil>
	driver_test.go:258: [34] values: [7a2f85497548f3db88542025e6f75908b8716c12-04 2346ad27d7568ba9896f1b7da6b5991251debdf2-04] location: 04 loc: 04 err: <nil>
	driver_test.go:258: [35] values: [7a2f85497548f3db88542025e6f75908b8716c12-05 2346ad27d7568ba9896f1b7da6b5991251debdf2-05] location: 05 loc: 05 err: <nil>
	driver_test.go:258: [36] values: [7a2f85497548f3db88542025e6f75908b8716c12-06 2346ad27d7568ba9896f1b7da6b5991251debdf2-06] location: 06 loc: 06 err: <nil>
	require.go:794: 
			Error Trace:	driver_test.go:255
			Error:      	Received unexpected error:
			            	Error: No usable addresses in the cluster
			            	requesting /schema
			            	github.com/pilosa/go-pilosa.(*Client).readSchema
			            		/home/jfontan/go/src/github.com/pilosa/go-pilosa/client.go:527
			            	github.com/pilosa/go-pilosa.(*Client).Schema
			            		/home/jfontan/go/src/github.com/pilosa/go-pilosa/client.go:315
			            	gopkg.in/src-d/go-mysql-server.v0/sql/index/pilosa.(*indexLookup).Values
			            		/home/jfontan/go/src/gopkg.in/src-d/go-mysql-server.v0/sql/index/pilosa/index.go:120
			            	gopkg.in/src-d/go-mysql-server.v0/sql/index/pilosa.TestPilosaHiccup
			            		/home/jfontan/go/src/gopkg.in/src-d/go-mysql-server.v0/sql/index/pilosa/driver_test.go:254
			            	testing.tRunner
			            		/usr/lib/go/src/testing/testing.go:777
			            	runtime.goexit
			            		/usr/lib/go/src/runtime/asm_amd64.s:2361
			Test:       	TestPilosaHiccup
=== RUN   TestDelete
--- FAIL: TestDelete (0.00s)
	require.go:794: 
			Error Trace:	driver_test.go:295
			Error:      	Received unexpected error:
			            	Error: No usable addresses in the cluster
			Test:       	TestDelete
```

--- a/sql/index/pilosa/driver_test.go
+++ b/sql/index/pilosa/driver_test.go
@@ -526,7 +526,7 @@ func newClientWithTimeout(timeout time.Duration) *pilosa.Client {
 func retry(ctx context.Context, fn func() error) error {
 	var (
 		backoffDuration = 200 * time.Millisecond
-		maxRetries      = 5
+		maxRetries      = 10
 
 		err error
 	)

